src/projects/TodoList/index.js

```diff
@@ -1,17 +1,40 @@
 import React from 'react';
 import style from './style.css';
 
-class TodoList extends React.Component {
-
+class Todo extends React.Component {
   render() {
-    /*
-      Before the return you can do all logic you need
-    */
     return (
-      <p>Hey im a todo list</p>
+      <p>{this.props.eachTodo}</p>
     )
   }
+}
```

Comment:
You can move this Todo component into its own file and import it - a good rule of thumb is to have a single component to a single file :)

Make sure in the new component file you have `export default Todo` at the bottom so you can export the component out of the file - then in this file you can do `import Todo from './Todo'`